README.rst

```diff
@@ -69,6 +69,14 @@ the Threema gateway. Run the following command to see usage information:
 
     $ threema-gateway --help
 
+Examples
+********
+
+You can find a few example scripts in the ``examples/`` directory.
+
+Note that most of them need to be adjusted to at least add your gateway ID
+credentials before they run successfully.
+
```

Comment:
Oh, sorry, what I actually meant was adding a note in the readme that the callback server listens on `http://<host>:<port>/gateway_callback` by default.
But this is good, too.

Reply:
I didn't touch the CLI :) Docs about the example scripts should probably be inside the example scripts, since people are going to copy them around.